docs/common/project/how-to-release.md

```diff
@@ -255,3 +260,60 @@ Java artifacts are available from Maven Central.
 
 Thanks to everyone for contributing!
 ```
+
+### Documentation Release
+
+Documentation and website needs to be updated as a part of Iceberg release.
+
+#### Iceberg version update
+
+Before release, publish a PR to update `landing-page/config.toml` and `docs/config.toml` with the new Iceberg version number.
+
+#### Documentation update
+
+To start the release process, run the following steps in the `iceberg-docs` repository to copy docs over 
+(assume the `iceberg-docs` repository and `iceberg` repository are in the same parent directory locally, and you are in the `iceberg` repository):
+
+```shell
+rm -rf ../iceberg-docs/docs/content/docs
+rm -rf ../iceberg-docs/landing-page/content/common
+cp -r docs/versioned ../iceberg-docs/docs/content/docs
+cp -r docs/common ../iceberg-docs/landing-page/content/common
+```
+
+This should be approved in a separated PR.
+
+#### Javadoc update
+
+In `iceberg` repository, generate the javadoc for your release and copy to the `javadoc` folder in `iceberg-docs` repo 
```

Comment:
s/iceberg repository/the iceberg repository
s/copy to the/copy it to the

Reply:
updated